contracts/test/integration/IntegrationTestVoltDeployment.t.sol

```diff
@@ -202,10 +205,16 @@ contract IntegrationTestVoltDeployment is DSTest, StdLib {
         uint256 endingUserVoltBalance = volt.balanceOf(address(this));
         uint256 endingPCVDepositFeiBalance = rariFEIPCVDeposit.balance();
 
-        assertEq(endingUserVoltBalance - startingUserVoltBalance, mintAmount);
-        assertEq(
-            (endingPCVDepositFeiBalance - startingPCVDepositFeiBalance),
-            (mintAmount - 1) /// goes down by 1 because of cToken pricing rounding down
+        assertApproxEq(
+            (endingUserVoltBalance - startingUserVoltBalance).toInt256(),
+            mintAmount.toInt256(),
+            0
+        );
```

Comment:
any reason we're changing these to `assertApproxEq` if the basis point deviation is set to 0?

Reply:
deleted this test, very flaky and not needed